app/api/routers/breeze_buddy.py

```
@@ -268,6 +270,43 @@ async def delete_outbound_number(
+                "order_id": order_id,
+                "updated_status": "FINISHED",
+                "updated_outcome": "ABORT"
+            }
+        else:
+            return {
+                "status": "not_found",
+                "message": f"No pending call found for lead {abort_request.lead_id}",
+                "lead_id": abort_request.lead_id
+            }
+            
+    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)